DependencyInjection/Configuration.php

```diff
@@ -85,6 +85,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('enable_sibling_provider')->defaultValue(false)->end()
                     ->end()
                 ->end()
+                ->arrayNode('content_listener')
+                    ->canBeDisabled()
```

Comment:
While we agree, I just wanted to point out that you can easily keep BC by keeping content_listener.enabled and use_content_listener in sync in case we want to add more config in the future.